image/base/slim.Dockerfile

```
@@ -122,5 +122,4 @@ COPY sbin/ /sbin
 # Discourse specific bits
 RUN useradd discourse -s /bin/bash -m -U &&\
     install -dm 0755 -o discourse -g discourse /var/www/discourse &&\
-    sudo -u discourse git clone --depth 1 https://github.com/discourse/discourse.git /var/www/discourse &&\
-    sudo -u discourse git -C /var/www/discourse remote set-branches --add origin tests-passed
+    sudo -u discourse git clone --filter=tree:0 https://github.com/discourse/discourse.git /var/www/discourse
```

```suggestion
    sudo -u discourse git clone --filter=tree:0 https://github.com/discourse/discourse /var/www/discourse
```

I believe `.git` suffix is just a back-compat redirect, no need to keep that 😃

---

GitHub shows that as clone URL, so I assume that's the recommended URL. I think the redirect happens only for regular browsers, not for the git client.